### ISSUE TITLE
The pretrained Graph U-Net model can not give same results in the paper

### ISSUE THREAD

**Issue by Moddel** [bardiadoosti/HOPE#27]
Hi! @bardiadoosti 
Thank you for your excellent work
I test the pretrained Graph U-Net model from http://vision.soic.indiana.edu/wp/wp-content/uploads/graphunet.tar.gz on FPHA dataset by feeding the GT 2d coordinates. I got the following results:
test mse loss: 702.49
test mean distance error(on 29 keypoints): 28.06 mm

However, in the Tabel 1&2 of the paper, the test mean distance error is 6.81mm. I am really confused about the large gap.
Looking for you reply and thank you again.

**Comment by bardiadoosti** [comment]
Hi,

Yes error should be around 20. Because of a small change in the source code the weights was incompatible with the current model and I had to train the model again so to quickly provide the weights I just trained the model with 2K iterations instead of 10K. I will complete the train cycle as soon as I got a free time and I had free GPUs.
